.github/CODEOWNERS

```diff
@@ -965,6 +965,11 @@
 # ServiceLabel: %Nginx %Mgmt
 # ServiceOwners:                                                   @jamesfan1 @bangbingsyb @spencerofwiti
 
+# ServiceLabel: %Private Mobile Network %Mgmt
+# ServiceOwners:                                                   @sathiyan-sivathas @richardwhiuk
+
+# PRLabel: %Mobile Network
+/sdk/mobilenetwork/Azure.ResourceManager.*/												 @tompretty
```

Comment:
```suggestion
# PRLabel: %Network - Mobile
/sdk/mobilenetwork/Azure.ResourceManager.*/												 @tompretty

# ServiceLabel: %Network - Mobile %Mgmt
# ServiceOwners:                                                   @sathiyan-sivathas @richardwhiuk
```

Added `Network - Mobile` to follow existing network-related label names and reformatted to match existing patterns.

- Please move to L962 to preserve alphabetical ordering.

- The support owners that you're adding do not have the correct organization memberships or permissions in the repository.   They'll need to follow the onboarding docs linked above to be used as CODEOWNERS.

- Your account has the correct memberships but still lacks the necessary write permissions.

I'll check back on permissions tomorrow, as they'll often take a day to fully propagate.   In the meantime, feel free to engage with Arthur's team for review of the actual library code.

Reply:
Okay, I've renamed to `%Network - Private Mobile Network` as I think `Private Mobile Network` is what people in my team will be expecting to look for (ie that's our service tree name), but it's now following the naming convention. 

I've listed myself as the owner for now as it doesn't seem anyone else in my team has done the onboarding yet. I'll follow up with the team to get proper owners sorted out.

I've also checked again with the `.\Validate-AzsdkCodeOwner.ps1` script and it's now detecting that I have all the required permissions.

Reply:
Okay I've actually updated both to be the `@Azure/private-5g-core` team instead

Reply:
Looks like that team doesn't have the required permissions, I've requested they be added in the teams channel:

https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1717151444475?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1717151444475&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1717151444475

Reply:
Since the team does not currently own this service and the new owner has not been determined, let’s just proceed with merging this PR and maintain the current state.